examples/pytorch/image-classifier-resnet50/README.md

```diff
@@ -8,7 +8,7 @@ This example implements an image recognition system using ResNet50, which allows
 
 There are 3 Cortex APIs available in this example:
 
-1. [cortex.yaml](cortex.yaml) - to be used with any instances.
+1. [cortex_cpu.yaml](cortex_cpu.yaml) - can be used with any instances.
```

Comment:
I agree for a slightly different reason. The examples should just work with `cortex deploy` so not having a cortex.yaml would prevent this.

Reply:
I changed it because when I wanted to run the inferentia example, I accidentally ran `cortex deploy` which deployed the CPU one, so I was thinking forcing you to be explicit might be nice. But that makes sense too, so I'll put it back